apps/style/common.scss

```diff
@@ -1201,11 +1201,9 @@ $workspace-header-button-margin: ($workspace-headers-height - $workspace-header-
   .goog-menuitem-hover {
     background-color: rgb(255 255 255 / 0.35);
   }
-  &:not(.blocklyGridDropdownMenu) {
-    border-color: rgb(84 84 84) !important;
-  }
+  border-color: rgb(84 84 84) !important;
   &.blocklyGridDropdownMenu {
-    background-color: white !important;
+    background-color: white;
```

Comment:
Is this rule now necessary? Seems like it's overridden.

Reply:
I wasn't sure if there might be some case where it's still used if we somehow don't use another color later on. It's probably safe either way?

Re: specific colors, I'd say that's out of scope for this fix and I don't recall ever seeing tickets about it in the past. Once we move Sprite Lab to Google Blockly, this would be determined by the style's `secondaryColour` prop which would give us control to tweak it how we like.

Reply:
Sounds good - thanks!